msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -1,60 +1,134 @@
-> * Bots must use the `ChannelMessage.Read.Group` RSC permission appropriately to build and enhance engaging experience for users in the team or they will not pass the store approval. The app description must include how the bot uses the data it reads.
-> * The `ChannelMessage.Read.Group` RSC permission may not be used by bots as a way to extract large amounts of customer data.
+> * Services that need access to all Teams message data must use the Graph APIs that  provide access to archived data in channels and chats.
+> * Bots must use the `ChannelMessage.Read.Group` and `ChatMessage.Read.Chat` RSC permission appropriately to build and enhance engaging experience for users in the team or they will not pass the store approval. The app description must include how the bot uses the data it reads.
+> * The `ChannelMessage.Read.Group` and `ChatMessage.Read.Chat` RSC permission may not be used by bots to extract large amounts of customer data.
 
 ## Update app manifest
 
 For your bot to receive all channel messages, RSC must be configured in the Teams app manifest with the `ChannelMessage.Read.Group` permission specified in the `webApplicationInfo` property.
 
-:::image type="content" source="~/bots/how-to/conversations/Media/appmanifest.png" alt-text="Screenshot of app manifest update.":::
+For more information about manifest schema, see [app manifest schema for Teams](../../../resources/schema/manifest-schema.md).
 
-The following is an example of the `webApplicationInfo` object:
+:::image type="content" source="../../../assets/images/bots/RSC/appmanifest_1.png" alt-text="app manifest":::
 
-* **id**: Your Microsoft Azure Active Directory (Azure AD) app ID. This can be the same as your bot ID.
-* **resource**: Any string. This field has no operation in RSC, but must be added and have a value to avoid error response.
-* **applicationPermissions**: RSC permissions for your app with `ChannelMessage.Read.Group` must be specified. For more information, see [resource-specific permissions](/microsoftteams/platform/graph-api/rsc/resource-specific-consent#resource-specific-permissions).
+The following list describes the `webApplicationInfo` object:
 
-The following code provides an example of the app manifest:
+* **webApplicationInfo.id**: Your Microsoft Azure Active Directory (AAD) app ID. The app ID can be the same as your bot ID.
+* **webApplicationInfo.resource**: Any string. The resource field has no operation in RSC, but must be added with a value to avoid error response.
+* **authorization.permissions.resourceSpecific**: RSC permissions for your app with either or both `ChannelMessage.Read.Group` and `ChatMessage.Read.Chat`must be specified and it depends on the conversation contexts relevant to your app.  For more information, see [resource-specific permissions](/microsoftteams/platform/graph-api/rsc/resource-specific-consent#resource-specific-permissions).
```

Comment:
Not sure why we rewrote this from the last version. Can we please update with the content below to simplify:

**authorization.permissions.resourceSpecific**: RSC permissions for your app with either or both `ChannelMessage.Read.Group` and `ChatMessage.Read.Chat` specified. For more information, see [resource-specific permissions](/microsoftteams/platform/graph-api/rsc/resource-specific-consent#resource-specific-permissions).